cli/internal/cloudcmd/iam.go

```diff
@@ -19,26 +19,26 @@ import (
 	"github.com/edgelesssys/constellation/v2/cli/internal/terraform"
 	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
 	"github.com/edgelesssys/constellation/v2/internal/cloud/gcpshared"
```

Comment:
same here

Reply:
The goal here was to align the usage of the cloudcmd IAM Terraform clients with the cloud resource terraform client.
Again, this shouldn't have undone any recent changes, but simply simplified logic in some places and moved the Terraform client creation from the constructor to the methods of the client (Similar to the cloud resource creator).